### PR TITLE
feat(starr-anime): add several RlsGrps to LQ

### LINKED ISSUE #1995: [Request] (Anime): Evaluate addition of Moozzi2 to Anime LQ Groups
Probably should be yes

Origin user: fonduemangvi
Origin message: [Discord Server Link](https://discord.com/channels/492590071455940612/934222763709833217/1257080950274261123)
Reporting user: dascactus
Reporting message: [Discord Server Link](https://discord.com/channels/492590071455940612/934222763709833217/1257083202930217013)

### PR DESCRIPTION
**Pull request by nuxencs**
# Pull Request

## Purpose

Resolve #1995 

## Approach

Add `Mozzi2`, `uP`, `Kallango`, `Emmid` and `Sokudo` to the `Anime LQ Groups` CF.

- Sokudo - AV1
- Emmid - some releases missing subtitles, only have dubtitles, includes many unnecessary language tracks, not consistent with track labelling and order
- Kallango/uP - Not on Nyaa and I cannot figure out why I initially banned them. My guess though is that both do upscales which replace older shows. Kallango seems to be exclusively on one of the unnameables, but they have some non-anime releases on others where it is just pt-BR releases.
- Moozzi2 - As it was getting uploaded to usenet, was replacing a bunch of people's releases with low quality raws.

## Requirements

- [x] These changes meet the standards for [contributing](https://github.com/TRaSH-Guides/Guides/blob/master/CONTRIBUTING.md).
- [x] I have read the [code of conduct](https://github.com/TRaSH-Guides/Guides/blob/master/.github/CODE_OF_CONDUCT.md).

### CODE REVIEW

File: docs/json/radarr/cf/anime-lq-groups.json [hunk 1]
```diff
@@ -501,6 +519,15 @@
         "value": "\\b(Kanjouteki)\\b"
       }
     },
+    {
+      "name": "KazeTV",
```
Comment (rg9400) [inline docs/json/radarr/cf/anime-lq-groups.json:523]: not a big deal, but this should be all caps (KAZETV). Rest looks good to me.

Comment (nuxencs) [reply docs/json/radarr/cf/anime-lq-groups.json:523]: will change that 👍🏼

**Review by rg9400 — DISMISSED** [review]
lgtm

**Review by rg9400 — DISMISSED** [review]

**Comment by rg9400** [comment]
Copying from Discord for rationale on the groups:
Sokudo - AV1
Emmid - some releases missing subtitles, only have dubtitles, includes many unnecessary language tracks, not  consistent with track labelling and order
Kallango/uP - Not on Nyaa and I cannot figure out why I initially banned them. My guess though is that both do upscales which replace older shows. Kallango seems to be exclusively on one of the unnameables, but they have some non-anime releases on others where it is just pt-BR releases.
Moozzi2 - As it was getting uploaded to usenet, was replacing a bunch of people's releases with low quality raws.

**Review by TRaSH- — APPROVED** [review]
Looks good to me. I added the reasons why they were added to the Anime LQ to the PR description.

**Pull request merged**